hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/api/db.json

```diff
@@ -742,55 +742,55 @@
         "leaderNode": "localhost1.storage.enterprise.com",
         "datanodes": [
           {
-          "level": 0,
-          "parent": null,
-          "cost": 0,
-          "uuid": "9e4e92da-4abd-461d-b042-dc4fe72e8727",
-          "uuidString": "9e4e92da-4abd-461d-b042-dc4fe72e8727",
-          "ipAddress": "172.22.0.5",
-          "hostName": "ozone_datanode_1.ozone_default",
-          "ports": [
-            {
-              "name": "REPLICATION",
-              "value": 9886
-            },
-            {
-              "name": "RATIS",
-              "value": 9858
-            },
-            {
-              "name": "RATIS_ADMIN",
-              "value": 9857
-            },
-            {
-              "name": "RATIS_SERVER",
-              "value": 9856
-            },
-            {
-              "name": "RATIS_DATASTREAM",
-              "value": 9855
-            },
-            {
-              "name": "STANDALONE",
-              "value": 9859
-            }
+            "level": 0,
+            "parent": null,
+            "cost": 0,
+            "uuid": "9e4e92da-4abd-461d-b042-dc4fe72e8727",
+            "uuidString": "9e4e92da-4abd-461d-b042-dc4fe72e8727",
+            "ipAddress": "172.22.0.5",
+            "hostName": "ozone_datanode_1.ozone_default",
```

Comment:
Could you please revert these changes from the code.
The changes show only tab spaces being added.

Reply:
These can be reverted right @smitajoshi12